### PR TITLE
Tello-603 Document ability to seed Kibana dashboard for admins

### PR DESCRIPTION
**Pull request by bmcelvee**
Trello: 
https://trello.com/c/NI2amjYd/603-document-ability-to-seed-kibana-dashboard-for-adminsloggingepic-ois-agl-exp

https://trello.com/c/O6MCrGUx/454-3-ability-to-seed-kibana-dashboard-for-adminsloggingepic-ois-agl-exp

### CODE REVIEW

**Comment by bmcelvee** [comment]
@jcantrill PTAL

**Comment by bmcelvee** [comment]
@tmorriso-rh Please peer review 😸

**Review by jcantrill — APPROVED** [review]

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Added `openshift_logging_elasticsearch_kibana_index_mode` parameter to the
xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables] section.
%

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -413,6 +413,14 @@ these instances on nodes reserved or optimized for running them.
 For example, the selector could be `node-type=infrastructure`. At least
 one active node must have this label before Elasticsearch will deploy.
 
+|`openshift_logging_elasticsearch_kibana_index_mode`
+|The default value, `unique`, allows users to each have their own Kibana index. In
```
Comment (bmcelvee) [inline install_config/aggregate_logging.adoc]: Test

**Pull request merged**